### DISCUSSION TITLE
Is it possible to limit searching specific domains?

### DISCUSSION THREAD

**Discussion by TheNomad11** [benbusby/whoogle-search#435] [Q&A]
I wonder if I can use Whoogle as a search engine for my website(s). Is it possible to define the domain(s) that Whoogle searches? Either via whoogle confoguration or the embed code? Thanks!

**Comment by alefvanoon** [comment]
Hmmm, yes you can.(Just like Google)

search for example on domain.com
example site:domain.com

I can't say it works great but it works.

**Reply by TheNomad11** [reply]
It does work, but I did not explain good enough. I would like to replace the google search form on my websites with a more privacy friendly one. So: I just installed Whoogle on a vps. Can I embed the Whoogle search form on my website so that site visitors can use it to search for content on my website?

**Reply by DUOLabs333** [reply]
It should be possible -- just make sure to append `site:<whatever your site is>` to the end of every request (you can probably modify the code to do that)
